app/filters.php

```diff
@@ -22,6 +22,14 @@
     if (is_singular('pcc-event') && get_post()->post_parent) {
         $classes[] = 'pcc-event-session';
     }
+    /** Add project class to project pages */
+    $template = [];
+    preg_match('/([a-z]*-[a-z]*-[a-z]*)/', get_page_template_slug(), $template);
+    if ($template[0] == "page-project-plain" || $template[0] == "single-pcc-project") {
+      $classes[] = 'project';
```

Comment:
My code editor is giving me a PHP formatting error for this (it doesn't match the specified PHP_Codesniffer indentation rules). Not a big deal, but can be addressed by running:

```bash
composer standards
```

(To see the listed issues.)

```bash
vendor/bin/phpcbf
```

To automatically fix them.